Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,17 @@
+Actors: Regular User, Artist, Admin
+
+Regular User - Draw on Digital Prosthetic, Browse Gallery, Contact Artists
+A regular user would use this app for fun, to enhance their creativity by drawing on the flattened surface of a prosthetic, or simply to browse the galleries without actually visiting the website.
+This would be a fun experience and ensure that the galleries would have a higher visibilty by being exposed to more individuals through the app.
+Amputees would fall under this category as well.
+An amputee might not necessarily use this app to look for prosthetics or artists, they could use this for fun as well, however, artworks could have their author's contact details displayed in the app, so
+an amputee would know who to contact in case they're interested in prosthetic. This could be their main platform (along with the website) for finding unique prosthetics made by professional artists.
+
+Artist - Draw on Digital Prosthetic, Browse Gallery, Post Artwork to Promotional Gallery
```

Comment:
Would posting art for an artist really happen from a mobile app? Also, I think the admin is the only one to either accept/decline artists and that's how their artwork ends up in the gallery. We don't allows posting, because we don't have users.

Reply:
Reworked this bit so everyone can post an image from the app to the "Public Gallery" but images can only be uploaded to the "Promotional Gallery" from the website AND ONLY by the admin.

Artists can only apply to be an "Artist", the rest is up to the admin.